.circleci/config.yml

```diff
@@ -14,7 +14,7 @@ jobs:
 
   build-and-test:
     macos:
-      xcode: 11.6.0 # Specify the Xcode version to use
+      xcode: 12.0.1 # Specify the Xcode version to use
```

Comment:
You'll need to update the iOS simulator version as well. It looks like "iPhone 8 (13.5)" is a valid simulator in the 12.0.1 image.
Here's the AEPAudience CircleCI build which lists the simulators for the 12.0.1 image:
https://app.circleci.com/pipelines/github/adobe/aepsdk-audience-ios/218/workflows/23e1af05-5c85-4455-97ab-845dbdefc192/jobs/225?invite=true#step-111-74